### ISSUE TITLE
Improve display of markers for zoomed out maps

### ISSUE THREAD

**Issue by op-hub** [amosproj/amos2024ss04-building-information-enhancer#199]
## User story
1. As a User
2. I want Zoomed out markers to merge into one
3. So that the map is not so crowded

## Acceptance criteria
* Different marker merge into one marker
* Not to much overlap between markers

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release

**Comment by Lucas-Nan** [comment]
currently working on - working for single map but bad with pinned maps
